ocs_ci/utility/rosa.py

```diff
@@ -71,20 +71,38 @@ def create_cluster(cluster_name, version, region):
     cluster_type = config.ENV_DATA.get("cluster_type", "")
     provider_name = config.ENV_DATA.get("provider_name", "")
     rosa_mode = config.ENV_DATA.get("rosa_mode", "")
+    private_link = config.ENV_DATA.get("private_link", "")
+    machine_cidr = config.ENV_DATA.get("machine-cidr", "10.0.0.0/16")
     cmd = (
         f"rosa create cluster --cluster-name {cluster_name} --region {region} "
-        f"--compute-nodes {compute_nodes} --compute-machine-type "
-        f"{compute_machine_type}  --version {rosa_ocp_version} {multi_az}--sts --yes"
+        f"--machine-cidr {machine_cidr} --compute-nodes {compute_nodes} "
+        f"--compute-machine-type {compute_machine_type} "
+        f"--version {rosa_ocp_version} {multi_az}--sts --yes"
     )
     if rosa_mode == "auto":
         cmd += " --mode auto"
-    if cluster_type.lower() == "consumer" and config.ENV_DATA.get("provider_name", ""):
+
+    if (
+        config.ENV_DATA["appliance_mode"]
+        and config.ENV_DATA.get("cluster_type", "") == "consumer"
+    ):
+        subnet_ids = config.ENV_DATA["ms_provider_subnet_ids_per_region"][region][
```

Comment:
Where we define the parameter `ms_provider_subnet_ids_per_region`?

Reply:
it is in credential_conf-  ocm-credentila-stage and ocm-credentials-prod file defined by MR https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/merge_requests/1053

Reply:
Okay, thanks.